libs/log/logger.go

```diff
@@ -2,29 +2,57 @@ package log
 
 import (
 	"io"
+	"sync"
+)
+
+const (
+	// LogFormatPlain defines a logging format used for human-readable text-based
+	// logging that is not structured. Typically, this format is used for development
+	// and testing purposes.
+	LogFormatPlain string = "plain"
+
+	// LogFormatText defines a logging format used for human-readable text-based
+	// logging that is not structured. Typically, this format is used for development
+	// and testing purposes.
+	LogFormatText string = "text"
+
+	// LogFormatJSON defines a logging format for structured JSON-based logging
+	// that is typically used in production environments, which can be sent to
+	// logging facilities that support complex log parsing and querying.
+	LogFormatJSON string = "json"
 
-	kitlog "github.com/go-kit/kit/log"
+	// Supported loging levels
+	LogLevelDebug = "debug"
+	LogLevelInfo  = "info"
+	LogLevelWarn  = "warn"
+	LogLevelError = "error"
+	LogLevelFatal = "fatal"
```

Comment:
Do we support warn and fatal?

Reply:
We don't technically, although I think we should support warn at the least. What do you think? I'll remove fatal.

Reply:
Makes sense, do you need to update the interface to expose it as well?